src/digitalmarketplace/components/alert/template.njk

```diff
@@ -0,0 +1,10 @@
+<div data-module="dm-alert" class="dm-alert dm-alert--{{ params.type }}{%- if params.classes %} {{ params.classes }}{% endif %}"{%- for key, value in params.attributes %} {{key}}="{{value}}"{% endfor %}>
+    <h{% if params.headingLevel %}{{ params.headingLevel }}{% else %}2{% endif %} class="dm-alert--title">
+        {{ params.titleHtml|safe if params.titleHtml else params.titleText }}
+    </h{% if params.headingLevel %}{{ params.headingLevel }}{% else %}2{% endif %}>
+    {% if params.html or params.text %}
+        <div class="dm-alert--body">
```

Comment:
I think the `dm-alert--body` should be `dm-alert__body`. The body is an element of the component not a modification of the component/element.